app/src/main/java/ar/edu/unlam/mobile/scaffold/ui/screens/SecondSreen.kt

```diff
@@ -0,0 +1,86 @@
+package ar.edu.unlam.mobile.scaffold.ui.screens
+
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.* // ktlint-disable no-wildcard-imports
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.material3.* // ktlint-disable no-wildcard-imports
+import androidx.compose.runtime.* // ktlint-disable no-wildcard-imports
```

Comment:
https://www.jetbrains.com/help/idea/creating-and-optimizing-imports.html#disable-wildcard-imports